implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -41,6 +41,32 @@ pub struct CliState {
     database: SqlxDatabase,
     application_database: SqlxDatabase,
     tracing_enabled: TracingEnabled,
+    tx: Option<tokio::sync::broadcast::Sender<ChannelMessageType>>,
+}
+
+const DEFAULT_CHANNEL_CAPACITY: usize = 16;
+pub type ChannelMessageType = String;
```

Comment:
This should be an enum eventually, and `exit` one of the cases.

Reply:
agreed; also removed "exit", since im not closing channels anymore